modules/server/src/main/scala/seqexec/server/gems/Gems.scala

```diff
@@ -140,7 +144,7 @@ object Gems {
   def fromConfig[F[_]: MonadError[*[_], Throwable]](
```

Comment:
Maybe we put `Logger` in the `F` constraints?

Reply:
Also, there's the `MonadThrow` alias now.